flow.sh

```
@@ -23,4 +23,22 @@ if [[ "$TESTS_FILE" == "" ]]; then
+# Add test filter support
+TEST_FILTER=()
+if [[ "$TEST" != "" ]]; then
+    TEST_FILTER=(-t "$TEST")
+fi
+
+# To run specific test files, use:
+# TEST="tests/flow/test_function_calls:testFunctionCallsFlow.test89_JOIN" FAIL_FAST=1 ./flow.sh
+# To run all tests in a specific file, use:
+# TEST="tests/flow/test_function_calls" FAIL_FAST=1 ./flow.sh
+
+RLTest ${TEST_FILTER[@]:--f "$TESTS_FILE"} --module "$TARGET_DIR/$TARGET" --no-progress $PARALLELISM $STOP_ON_FAILURE --clear-logs --log-dir tests/flow/logs $V
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)